icu4c/source/common/unicode/uconfig.h

```diff
@@ -45,6 +45,11 @@
  * @stable ICU 2.4
  */
 
+// RTC defines
+#define U_COMMON_IMPLEMENTATION // build the icu common library
+#define U_STATIC_IMPLEMENTATION // build icu as a static library
+// end RTC defines
```

Comment:
Would it be better to place these into the LUA file?

Reply:
That's where they were before but then every consumer of icu would need to define these as well which gets a bit bloated.  Adding to the uconfig file will ensure that consumers will get the right configurations since ICU has this config file for that purpose.